app/src/openhpi/res/values/strings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="settings_copyright">&#169; 2012 &#8211; %d Hasso Plattner Institute</string>
+
+    <string name="login_sso" translatable="false">HPI Identity</string>
```

Comment:
The openHPI app shows `SSO` in german, because `login_sso` is defined as `SSO` in the german `main` flavor, which is used as fallback and not this english variant.

Reply:
I fixed that for all flavors in https://github.com/openHPI/xikolo-android/commit/bccf4027b686c18fb45dd42ad306f391a59b7a08 and removed the usage of `translatable="false"` for all flavor strings. This should give us lint issues in case we miss that in the future.